### ISSUE TITLE
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 31: ordinal not in range(128)

### ISSUE THREAD

**Issue by khenderick** [openvstorage/framework#1039]
```
...
  File "/opt/OpenvStorage/ovs/extensions/generic/sshclient.py", line 207, in _clean_text
    return text.strip().replace(u'\u2018', u'"').replace(u'\u2019', u'"')
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 31: ordinal not in range(128)
```

**Comment by wimpers** [comment]
What is the context? When does this happen? Which use case? Maybe voldrv related?

**Comment by khenderick** [comment]
I don't remember when it happens, it's just a general remark that output containing special quotes are messing up our code. It's not volumedriver related.

**Comment by khenderick** [comment]
Fixed by #1049, packaged in openvstorage-2.7.4-rev.4161.5f26491

**Comment by JeffreyDevloo** [comment]
### Information
This error occurs when processing a byte string with characters that cannot be decoded.
Should have been resolved by the referenced pull request and further enhanced by https://github.com/openvstorage/framework/pull/1182.

We did not encounter this error anymore on our testing environments.

This scenario has also been covered by a unit test which expects all sorts of data being passed on.
```
root@ovs-node-2:~# ovs unittest /opt/OpenvStorage/ovs/extensions/generic/tests/test_sshclient



########################################################################################
# Processing test-module /opt/OpenvStorage/ovs/extensions/generic/tests/test_sshclient #
########################################################################################

test_text_cleanup (test_sshclient.SSHClientTest) ... ok

----------------------------------------------------------------------
Ran 1 test in 0.001s

OK



############
# OVERVIEW #
############

  - TestModule: /opt/OpenvStorage/ovs/extensions/generic/tests/test_sshclient  (1 test)
    - DURATION: < 1 second
    - SUCCESS: 1

```
